source/c_dataexchange.rst

```diff
@@ -0,0 +1,406 @@
+
+Time series can be uploaded through a 4 column csv.
+Select both the organisation you want to upload and the asset type the time series belongs to (e.g. groundwater station).
+The csv should not contain a header.
+
+.. csv-table:: Example with headers
+    :header: timestamp, unit id / name, value, asset id
+    
+    2015-01-01T00:00:00Z, GWmMSL, 248.0, your_own_code_1
+    2015-01-01T01:00:00Z, GWmBGS, 248.0, your_own_code_1
+    2015-01-01T00:00:00Z, GWmMSL, 252.3, your_own_code_2
+    2015-01-02T05:00:00Z, GWmMSL, 234.2, your_own_code_3
+
+The columns should contain:
+
+* **timestamp:** a timestamp in iso8601 format.
+* **unit id / name:** This is both the name of the observation type name and this will become the timeseries name.
+* **value:** value as either a float or integer number.
+* **asset id:** either an asset uuid or an organisation-code (an identifier for an asset, unique for your organisation). In case the assets have been added with a code under category [columns].
+
+Since a csv should not contain a header, your csv should look like this:
+
+.. csv-table:: Example without headers
+
+    2015-01-01T00:00:00Z, GWmMSL, 248.0, your_own_code_1
+    2015-01-01T01:00:00Z, GWmBGS, 248.0, your_own_code_1
+    2015-01-01T00:00:00Z, GWmMSL, 252.3, your_own_code_2
+    2015-01-02T05:00:00Z, GWmMSL, 234.2, your_own_code_3
+
+Authentication
+--------------
+
+SFTP users are authenticated with a username / password.
+
+Supported data formats
+----------------------
+
+Via SFTP the following formats are supported:
+
+    * CSV
+    * PiXML
+
+Every supplier has its own directory on the SFTP.
+It can be accessed by logging in with the Lizard credentials.
+
+As soon as a new file is uploaded to the SFTP server, it will be automatically recognised and processed by Lizard.
+After processing the file is moved to a backup for a limited period of time.
+
+When a file is rejected, the supplied file is moved to the directory ‘rejected’ and a message is sent to the suppliers Inbox.
+In the Inbox a supplier can see the status of his supplied files.
+
+CSV
++++++
+
+Use CSV for supplying timeseries data with numerical or textual values according to the following format:
+
+.. code-block:: none
+
+    <timestamp>,<timeseries_supplier_code or uuid>,<value>[\n]
+    <timestamp>,<timeseries_supplier_code or uuid>,<value>[\n]
+    <timestamp>,<timeseries_supplier_code or uuid>,<value>[\n]      
+    …
+
+Where:
+
+    * **timestamp:** time in UTC in ISO 8601 format. For example 2012-10-26T09:22:35Z. Supplying timestamps in different timezone is only allowed when the UTC offset is added to the timestamp according to ISO 8601. For example: 2012-10-26T07:22:35+02.
+    * **timeseries_supplier_code or UUID:** supplier_code attribute of timeseries as registered by administrator/supplier or the UUID of the timeseries object.
+    * **value:** numerical or textual value.
+    * **[\\n]:** newline character.
```

Comment:
Spaties voor bullets weg.
Geldt ook voor de rest van het bestand. Dit is tot nu toe de enige waarin ik het gezien heb.